build.yaml

```diff
@@ -51,6 +51,7 @@ BuildWorkflow:
     - Build_Module_ModuleBuilder
     - Build_NestedModules_ModuleBuilder
     - Create_changelog_release_output
+    - Generate_help_from_built_module
```

Comment:
We (or maybe only I) use "build" many times during the development and I like it because it is very fast (about 4s). Can we move this step into separate one (new) or to "pack" command? Otherwise building process takes now 35-60s.